nix/nixos/tests/default.nix

```diff
@@ -0,0 +1,20 @@
+{ pkgs
+, supportedSystems ? [ "x86_64-linux" ]
+}:
+
+with pkgs;
+with pkgs.commonLib;
+
+ let
+  forAllSystems = genAttrs supportedSystems;
+  importTest = fn: args: system: let
+    imported = import fn;
+    test = import (pkgs.path + "/nixos/tests/make-test-python.nix") imported;
```

Comment:
looks like this trigger
```
job ‘nixosTests.basicTest.x86_64-linux’:
error: access to path '/nix/store/nix/nixos' is forbidden in restricted mode
```
in [hydra](https://hydra.iohk.io/jobset/Cardano/cardano-db-sync-pr-950#tabs-errors)...